packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -1697,19 +1707,36 @@ class Viewport implements IViewport {
    *        * The viewRef must be applicable to the current stack or volume, otherwise an exception will be thrown
    * @param viewPres - the presentation information to apply to the current image (as chosen above)
    */
-  public setView(viewRef?: ViewReference, viewPres?: ViewPresentation) {
-    if (viewPres) {
-      const { displayArea, zoom = this.getZoom(), pan, rotation } = viewPres;
-      if (displayArea !== this.getDisplayArea()) {
-        this.setDisplayArea(displayArea);
-      }
-      this.setZoom(zoom);
-      if (pan) {
-        this.setPan(vec2.scale([0, 0], pan, zoom) as Point2);
-      }
-      if (rotation >= 0) {
-        this.setRotation(rotation);
-      }
+  public setView(viewRef: ViewReference, viewPres: ViewPresentation) {
```

Comment:
I thought we removed this no? 
I'm fine with calling those separately, if you are too remove this extra method